crypto.go

```diff
@@ -237,8 +237,8 @@ func onionEncrypt(sharedSecret *Hash256, data []byte) []byte {
 
 // minOnionErrorLength is the minimally expected length of the onion error
 // message. Including padding, all messages on the wire should be at least 256
-// bytes. We then add the size of the sha256 HMAC as well.
-const minOnionErrorLength = 2 + 2 + 256 + sha256.Size
+// bytes.
+const minOnionErrorLength = 2 + 2 + 256
```

Comment:
Perhaps a new constant is needed for processing w/ and w/o this new behavior? Would allow us to eliminate the `+sha256.Size` below.

Reply:
Also IIUC, this is still valid for errors that don't include this new information.

Reply:
I don't think this commit changed the logic for the legacy errors. But good suggestion to add another constant, done.